spec/scenarios/meta_data_by_context_spec.rb

```diff
@@ -0,0 +1,130 @@
+describe 'querying Metadata by Context', type: :request do
+  %w{0 1 2}.each do |n|
+    context_key = "context_key_#{n}"
```

Comment:
What is it for?

Reply:
Do you mean context keys or the whole block?

Reply:
I think I get it, context_key variable - right? It's just shorter to define that variable than type "context_key_#{n}" again and again.